docs/docs-ref-autogen/officescript.yml

```diff
@@ -73,31 +54,18 @@ functions:
       the local machine.
     remarks: ''
 
-    isPreview: true
+    isPreview: false
     isDeprecated: false
     syntax:
-      content: 'export function downloadFile(fileProperties: FileProperties): void;'
+      content: >-
+        export function downloadFile(fileProperties: DownloadFileProperties):
+        void;
       parameters:
         - id: fileProperties
-          description: >-
-            The file to download.
-
-
-            **Throws**: `DownloadFileNameMissing` The error thrown if the
```

Comment:
These didn't get removed from the source. It looks like the preprocessor change you made in [PR 391](https://github.com/OfficeDev/office-scripts-docs-reference/pull/391/files) doesn't work with the new version, which is visible in [PR 403](https://github.com/OfficeDev/office-scripts-docs-reference/pull/403/files#diff-406cae059378fdab6c5becd5cb445514ddd7a827acf22b651bf3c6af0e388387)